integr8ly/integreatly-operator
# MGDAPI-1882 - Update/Create automated e2e tests/test cases (master based)

# Issue link
https://issues.redhat.com/browse/MGDAPI-1882

Current PR replaces old one: https://github.com/valerymo/integreatly-operator/pull/25 (oo-install based)

# What
Automation tests update for Observability Operator.
- Updated tests list: E01,E02,E03,E05,E09,"Verify Alerts are not firing...", A03,A05-A09,A13,A15, C01,C03,C04.


# Verification steps
- Check from command line that all updated Tests are passed.
- Check in pipeline that tests are passed.
